frontend/app/routes/$lang+/_public+/apply+/$id+/adult/living-independently.tsx

```diff
@@ -0,0 +1,165 @@
+
+import { ActionFunctionArgs, LoaderFunctionArgs, MetaFunction, json, redirect } from '@remix-run/node';
+import { useFetcher, useLoaderData, useParams } from '@remix-run/react';
+
+import { faChevronLeft, faChevronRight, faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useTranslation } from 'react-i18next';
+import { z } from 'zod';
+
+import pageIds from '../../../../page-ids.json';
+import { Button, ButtonLink } from '~/components/buttons';
+import { ErrorSummary, createErrorSummaryItems, hasErrors, scrollAndFocusToErrorSummary } from '~/components/error-summary';
+import { InputRadios } from '~/components/input-radios';
+import { Progress } from '~/components/progress';
+import { loadApplyAdultChildState, saveApplyAdultChildState } from '~/route-helpers/apply-adult-child-route-helpers.server';
+import * as adobeAnalytics from '~/utils/adobe-analytics.client';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { getFixedT } from '~/utils/locale-utils.server';
+import { getLogger } from '~/utils/logging.server';
+import { mergeMeta } from '~/utils/meta-utils';
+import { RouteHandleData, getPathById } from '~/utils/route-utils';
+import { getTitleMetaTags } from '~/utils/seo-utils';
+import { cn } from '~/utils/tw-utils';
+
+enum LivingIndependentlyOption {
+  No = 'no',
+  Yes = 'yes',
+}
+
+export type LivingIndependentlyState = `${LivingIndependentlyOption}`;
+
+export const handle = {
+  i18nNamespaces: getTypedI18nNamespaces('apply', 'apply-adult-child', 'gcweb'),
+  pageIdentifier: pageIds.public.apply.adultChild.livingIndependently,
+  pageTitleI18nKey: 'apply-adult-child:living-independently.page-title',
+} as const satisfies RouteHandleData;
+
+export const meta: MetaFunction<typeof loader> = mergeMeta(({ data }) => {
+  return data ? getTitleMetaTags(data.meta.title) : [];
+});
+
+export async function loader({ context: { session }, params, request }: LoaderFunctionArgs) {
+  const state = loadApplyAdultChildState({ params, request, session });
+  const t = await getFixedT(request, handle.i18nNamespaces);
+
+  const csrfToken = String(session.get('csrfToken'));
+  const meta = { title: t('gcweb:meta.title.template', { title: t('apply-adult-child:living-independently.page-title') }) };
+
+  return json({ id: state.id, csrfToken, meta, defaultState: state.adultChildState.livingIndependently });
+}
+
+export async function action({ context: { session }, params, request }: ActionFunctionArgs) {
+  const log = getLogger('apply/adult-child/living-independently');
+
+  const t = await getFixedT(request, handle.i18nNamespaces);
+
+  /**
+   * Schema for living independently.
+   */
+  const livingIndependentlySchema: z.ZodType<LivingIndependentlyState> = z.nativeEnum(LivingIndependentlyOption, {
+    errorMap: () => ({ message: t('apply-adult-child:living-independently.error-message.living-independently-required') }),
```

Comment:
Fix other places as well.

```suggestion
    errorMap: () => ({ message: t('apply-adult:living-independently.error-message.living-independently-required') }),
```